prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/ChannelCommandTest.java

```diff
@@ -191,7 +191,33 @@ public void testList() {
         int exitCode = commandLine.execute(CliConstants.Commands.CHANNEL, CliConstants.Commands.LIST,
                 CliConstants.DIR, dir.toString());
         Assert.assertEquals(ReturnCodes.SUCCESS, exitCode);
-        Assert.assertEquals(3, getStandardOutput().lines().filter(l->l.contains("manifest")).count());
+        Assert.assertEquals(3, getStandardOutput().lines()
```

Comment:
Use `Assertions.assertThat` and compare it to actual expected lines ("test1 g:a", "test2 g:a:v", "test3 file:/a:b")